### DISCUSSION TITLE
deepin-file-manager Edit file path URL after slow double click

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#3384] [Features Request & Ideas | 特性请求 & 头脑风暴]
With the same logic as renaming a file, the URL bar in File manager should be editable after two slow clicks at the address bar.

**Comment by BLumia** [comment]
I don't think that's a good idea since it is simply not a place to display file names. Make it not clickable is intended (v15 you can just click to edit) so people won't afraid to drag that place to move the window and also won't accidently trigger edit while trying to move the window.

The Edit option should be in the context menu already, and you can also use <kbd>Ctrl + L</kbd> to toggle edit which is a generic shortcut that can be used in many file manager apps :)
